### PR TITLE
Userid module: propagate ortb2.user.ext.eids to userIdsAsEids even if no UserId submodules

### PR DESCRIPTION
**Pull request by justadreamer**
## Type of change
- [x] Enhancement

## Description of change
Addresses https://github.com/prebid/Prebid.js/issues/12458 - we've spotted the issue that if there are no UserId submodules   added - the `bidRequest.userIdsAsEids` is empty, while some bid adapters peak into it instead of `ortb2.user.ext.eids`.  This PR addresses this issue by adding calling corresponding hook that would propagate eids into `bidRequest.userIdsAsEids` regardless of the submodules presence.

### CODE REVIEW

**Comment by justadreamer** [comment]
looks like there are some build failures this change causes - we are looking into them.

**Comment by ChrisHuie** [comment]
> looks like there are some test failures this change causes - we are looking into them.

It's on our end. Pr up to fix issue

**Comment by patmmccann** [comment]
Try merging in Master

File: modules/userId/index.js [hunk 1]
```diff
@@ -1211,6 +1223,10 @@ export function init(config, {delay = GreedyPromise.timeout} = {}) {
         initIdSystem({ready: true});
       }
     }
+    if (!addedUserIdHook && !startAuction.getHooks({hook: addUserIdsHook}).length) {
```
Comment (dgirardi) [inline modules/userId/index.js:1226]: I'd put it outside of the config listener - like [this](https://github.com/prebid/Prebid.js/commit/4ccf40e0b13976eb3b89baaaa7be0c46035a234f). Do you have a reason to prefer it here? I'm thinking that without any submodules, you wouldn't get any configuration either.

**Review by dgirardi — APPROVED** [review]

**Review by patmmccann — APPROVED** [review]

**Pull request merged**